src/routes/comments.js

```diff
@@ -0,0 +1,118 @@
+const express = require("express");
+const router = express.Router();
+const { body, validationResult } = require("express-validator");
+const Comment = require("../models/Comment");
+const User = require("../models/User");
+const mongoose = require("mongoose");
+
+//Comment Routes
+
+/*
+Route GET requests at root 
+Populate comment document with user document referenced in "user" field
+return response containing array of all comments in JSON on success
+*/
+router.get("/", async (req, res) => {
+  Comment.find()
+    .populate("user")
+    .exec()
+    .then(comments =>
+      res.json(
+        comments.map(comment => {
+          return {
+            _id: comment._id,
+            user: comment.user._id,
```

Comment:
The front end doesn't need to see the `user._id`, so initially we have the user be the user id, after we get the data from `populate()` we replace the id with the user name

Reply:
Okay got it, I fixed it so that user returns the username in the response, and deleted user._id